submissions/OlgaBarnikova/Popup/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+    box-sizing: border-box;     
+    list-style: none; 
+}
+
+a, a:active {
+    text-decoration: none;
+    color: rgb(94, 94, 226);
+}
+
+:focus {
```

Comment:
This is an invalid CSS selector: you can't set pseudoclass to nothing, you should provide an element:
```CSS
.block:focus {}

.a:focus {}

*:focus {}
```